docs/literate/make.jl

```diff
@@ -0,0 +1,89 @@
+using Literate: Literate
+using Test: @testset
+import Pkg
+
+# Create markdown and notebook files for `file`
+function create_files(title, file, repo_src, pages_dir, notebooks_dir; folder="")
+    notebook_filename = first(splitext(file)) * ".ipynb"
+    if folder != ""
```

Comment:
```suggestion
    if !isempty(folder)
```
This is not strictly necessary here, but it's best practice to use `isempty` instead of creating an empty container and comparing the contents.